backend/capellacollab/config/loader.py

```diff
@@ -11,34 +12,55 @@
 
 log = logging.getLogger(__name__)
 
-locations: list[pathlib.Path] = [
+config_locations: list[pathlib.Path] = [
     pathlib.Path(__file__).parents[2] / "config" / "config.yaml",
     pathlib.Path(appdirs.user_config_dir("capellacollab", "db"))
     / "config.yaml",
-    pathlib.Path("/etc/capellacollab/config.yaml"),
+    pathlib.Path("/etc/capellacollab") / "config.yaml",
 ]
 
-fallback_locations: list[pathlib.Path] = [
+config_fallback_locations: list[pathlib.Path] = [
     pathlib.Path(__file__).parents[2] / "config" / "config_template.yaml",
 ]
 
+json_schema_locations = [
```

Comment:
This is exactly one file and the path can be hardcoded (you can remove some logic, e.g., the for loop). 

Proposal: 

```py
def load_config_schema() -> dict:
    return json.load((pathlib.Path(__file__).parents[0] / "config_schema.json").open())
```